api/catalog/api/models/media.py

```diff
@@ -241,7 +241,10 @@ def save(self, *args, **kwargs):
         )
         if self.status != DEINDEXED:
             same_reports = same_reports.filter(reason=self.reason)
-        same_reports.update(status=self.status)
+
+        # Prevent redundant update statement when creating the report
+        if self.status != PENDING:
+            same_reports.update(status=self.status)
```

Comment:
Maybe this would be clearer if the status check was before the query on line 238 and caused an early return? Something like:

```py
if self.status == PENDING:
    return
```

Only because then it is quite clear that the intention is that pending reports should have no effect on other reports.